yGavrilovets/02_HTML/Python/HW1_Python1.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<HTML
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Python_1</TITLE>
+	</HEAD>
+	<BODY>
+		<TABLE width="100%">
```

Comment:
could you please remove this table, if you want to align picture, then use align attribute here, make changes in all same files with this logo-picture